package/MDAnalysis/analysis/contacts.py

```diff
@@ -455,6 +462,11 @@ def __init__(self, u, select, refgroup, method="hard_cut", radius=4.5,
                                                 box=self._get_box(refA.universe)))
                 self.initial_contacts.append(contact_matrix(self.r0[-1], radius))
 
+        if len(self.initial_contacts) > 0:
+            self.n_initial_contacts = self.initial_contacts[0].sum()
+        else:
+            self.n_initial_contacts = 0
```

Comment:
Could you please add a test for this `else` condition? It looks like you can trigger this condition if `refgroup=[]` is an empty list.

Reply:
- @lilyminium I noticed something that, my **```if```** condition checks for **```initial_contacts```**, which can never be of length 0, as it is always ```[array([], shape=(3341, 0), dtype=bool)]``` because of initialisation. So my **```else```** case may never get executed or might be i am wrong.
- So what can be done further from these options?
  1) If my explanation is correct then we can remove the **```else```** case from the **```contacts.py```** file.
  2) You can give me examples which initialises **```initial_contacts```** list as an empty list.

Reply:
Ok yeah, looking again at the code I don't think refgroup can be an empty list without running into errors earlier. In that case I agree, the else condition isn't necessary.